crates/context/README.md

```diff
@@ -0,0 +1,65 @@
+# Calimero Contexts
+
+- [Context Lifecycle](#context-lifecycle)
+- [Administrative Operations](#administrative-operations)
+- [Context State](#context-state)
+- [State Mutations](#state-mutations)
+- [Context Membership](#context-membership)
+  - [Invitations](#invitations)
+  - [Leaving](#leaving)
+- [Encryption](#encryption)
+- [Application Upgrade](#application-upgrade)
+- [Context Deletion](#context-deletion)
+
+A context in Calimero is an instance of a deployed application members of which share a synchronized state.
+
+A context is identified by a 32-byte key, which is randomly generated when the context is created. As of now, the key is not necessarily cryptographically secure, nor is it a dependency of cryptographic operations, but it is unique enough to mitigate collisions. Worth noting that this is likely to change in the future.
+
+All operations within a context performed by members will be broadcasted to all other members. This includes:
+
+- State mutations
+- Membership changes (invitation, leaving, kickouts)
```

Comment:
Do we have membership changes right now? If not, lets remove it

Reply:
Outside of the scope of what's here for the eventual shape, there's not much material to document about the current implementation of a context that is worthy of a doc, it'll all stay dynamic for a while

Unlike the other pieces

Reply:
So, do we have it right now or not? Lets not write things that will eventually come.

Reply:
In that case, there's no value to documenting this piece (yet). What we have now (context only, non-networking, non-storage) is barely half a page.

Reply:
Then let's have half a page. We must be transparent with what is available. The roadmap will answer what will come.

Reply:
removed